roles/nfs/tasks/server.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: install nfs server (redhat)
+  yum: "update_cache=yes name={{ item }}"
+  with_items:
+    - nfs-utils
+    - libnfsidmap
+  when: ansible_os_family == "RedHat"
+
+- name: install nfs server (debian)
+  apt: "update_cache=yes name={{ item }}"
+  with_items:
+    - nfs-kernel-server
+    - rpcbind
+  when: ansible_os_family == "Debian"
+
+- name: enable nfs server services (redhat)
+  service: "name={{item}} enabled=yes"
```

Comment:
thinking more, we should perhaps not enable these services as it will run them automatically at reboot. This will cause incorrect behavior if the node was decommissioned and cleaned-up earlier. wdyt?

As of now we don't enable (non-base) services by default as that ensures when a node get's rebooted (i.e. disappears from cluster), it can be brought back into cluster by cluster-manager which will ensure pushing the configuration as per the current cluster state.

Also, may be slightly unrelated if there are multiple masters, is it safe to start nfs-server on all nodes?